api/handlers/service_offering_test.go

```diff
@@ -271,4 +272,61 @@ var _ = Describe("ServiceOffering", func() {
 			})
 		})
 	})
+
+	Describe("DELETE /v3/service_offerings/:guid", func() {
+		JustBeforeEach(func() {
+			req, err := http.NewRequestWithContext(ctx, "DELETE", "/v3/service_offerings/offering-guid", nil)
+			Expect(err).NotTo(HaveOccurred())
+
+			routerBuilder.Build().ServeHTTP(rr, req)
+		})
+
+		It("deletes the service offering", func() {
+			Expect(serviceOfferingRepo.DeleteOfferingCallCount()).To(Equal(1))
+			_, actualAuthInfo, actualDeleteMessage := serviceOfferingRepo.DeleteOfferingArgsForCall(0)
+			Expect(actualAuthInfo).To(Equal(authInfo))
+			Expect(actualDeleteMessage.GUID).To(Equal("offering-guid"))
+			Expect(actualDeleteMessage.Purge).To(BeFalse())
+
+			Expect(rr).To(HaveHTTPStatus(http.StatusNoContent))
+		})
+
+		When("getting the service offering fails with not found", func() {
```

Comment:
The test verifies the error when deleting the offering fails with not found. Therefore, the text of this `When` should be `deleting the service offering fails with not found`